docker/alpine/Dockerfile

```diff
@@ -230,26 +199,21 @@ ENV GRASSBIN="/usr/local/bin/grass" \
 # https://proj.org/usage/environmentvars.html#envvar-PROJ_NETWORK
 ENV PROJ_NETWORK=ON
 
-# Add packages for fully enabling g.extension
-RUN apk add --no-cache $GRASS_RUN_PACKAGES
+## run simple LAZ test
+COPY docker/testdata/simple.laz /tmp/
+COPY docker/testdata/test_grass_session.py /scripts/
+COPY docker/alpine/grass_tests.sh /scripts/
+ENV GRASSBIN=grass
 
-# addon test in final stage: does g.extension install also C-extensions?
-# test raster C addon
-RUN grass --tmp-location XY --exec g.extension extension=r.gwr operation=add
-RUN grass --tmp-location XY --exec g.extension extension=r.gwr operation=remove -f
-# test vector C addon
-RUN grass --tmp-location XY --exec g.extension extension=v.centerpoint operation=add
-RUN grass --tmp-location XY --exec g.extension extension=v.centerpoint operation=remove -f
+# run some tests and cleanup
+RUN /bin/bash /scripts/grass_tests.sh && rm -f /scripts/grass_tests.sh \
+              /tmp/simple.laz /scripts/test_grass_session.py
```

Comment:
I just realized that these files were added in a previous stage, so removing them has no effect in image size, just more organized content of image. So I would still vote for removing them here.